dockerfile-image-update-cmdtool/src/main/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGithubUtil.java

```diff
@@ -117,7 +115,7 @@ public void modifyAllOnGithub(GHRepository repo, String branch,
 
     protected void modifyOnGithubRecursive(GHRepository repo, GHContent content,
                                            String branch, String img, String tag) throws IOException {
-        if (content.getType().equals(GITHUB_FILE)) {
+        if (content.getType().equals(Constants.GITHUB_FILE)) {
```

Comment:
Use `content.isFile()`

Reply:
Nice, this must be a new feature in with the library version bump. Fixed.